parsl/dataflow/dflow.py

```diff
@@ -197,8 +197,20 @@ def _create_task_log_info(self, task_id, fail_mode=None):
         task_log_info['task_inputs'] = str(self.tasks[task_id]['kwargs'].get('inputs', None))
         task_log_info['task_outputs'] = str(self.tasks[task_id]['kwargs'].get('outputs', None))
         task_log_info['task_stdin'] = self.tasks[task_id]['kwargs'].get('stdin', None)
-        task_log_info['task_stdout'] = self.tasks[task_id]['kwargs'].get('stdout', None)
-        task_log_info['task_stderr'] = self.tasks[task_id]['kwargs'].get('stderr', None)
+        stdout_spec = self.tasks[task_id]['kwargs'].get('stdout', None)
+        stderr_spec = self.tasks[task_id]['kwargs'].get('stderr', None)
+        try:
+            stdout_name, stdout_mode = get_std_fname_mode('stdout', stdout_spec)
+        except Exception as e:
+            stdout_name, stdout_mode = str(e), None
```

Comment:
this seems like a bad thing to do with an exception. Maybe it should just be allowed to propagate upwards?

Reply:
I just think it might be useful to catch this exception in monitoring db?

Reply:
probably add a logging line here?

Reply:
@benclifford Do you expect this exception to be raised at DFK?